### ISSUE TITLE
Emit calls on mutable flows not covered

### ISSUE THREAD

**Issue by sampengilly** [Kotlin/kotlinx-kover#147]
I've noticed some inaccurate coverage when using the (fairly common on Android) pattern of having a private mutable flow and a public immutable version of the same flow, e.g:

```
private val _theInternalFlow = MutableStateFlow(...)
val thePublicFlow: StateFlow<...> = _theInternalFlow
```

Lines that emit to the private flow are marked in yellow and don't contribute to the coverage count, despite the tests verifying emissions from the public flow. All surrounding lines of code are marked as green and included in coverage.

This is using the IntelliJ coverage engine (1.0.647) on Kover (0.5.0) and on Kotlin 1.5.21

**Comment by shanshin** [comment]
@sampengilly, hi, is it still reproduced in latest versions of IntelliJ Agent (e.g. `1.0.675`)?
We were unable to reproduce the problem, so if it repeats on the latest versions, could you send a small reproducer?

**Comment by shanshin** [comment]
Must be fixed in `0.6.0`, feel free to reopen if the problem reproduced again.
